src/io/parquet/write/primitive/basic.rs

```diff
@@ -98,26 +99,27 @@ where
     R: NativeType,
     T: num_traits::AsPrimitive<R>,
 {
+    let minmax = array
```

Comment:
Why not use the min/max from compute here?

Reply:
As they are primitive you could use `min_primitive`, `max_primitive` here.
I think the downside is having to enable the compute feature for writing parquet. Is that the reason it's not using the compute kernels @jorgecarleitao ?

Reply:
Indeed, maybe using `min_primitve` and `max_primitive` could have some advantages, to leverage SIMD for instance.

Reply:
> I think the downside is having to enable the compute feature for writing parquet. 

We could use a `[cfg(feature="compute")]` tag to use the kernels if compute is enabled and use iterators/ loops otherwise.

Reply:
That seems a good trade off to me.

Reply:
Although those compute kernels wouldn't be compatible with type that don't implement the `Simd` trait, `i128` for instance.